src/agent/agent.c

```diff
@@ -652,6 +652,48 @@ static AgentUnitInfo *agent_ensure_unit_info(Agent *agent, const char *unit) {
         return info;
 }
 
+static int passthrough_agent_method_cb(sd_bus_message *m, void *userdata, UNUSED sd_bus_error *ret_error) {
+        _cleanup_systemd_request_ SystemdRequest *req = userdata;
+
+        if (sd_bus_message_is_method_error(m, NULL)) {
+                /* Forward error */
```

Comment:
Nit-picky: I think this comment could be removed.

Reply:
Done